en/document-summaries.html

```diff
@@ -6,117 +6,220 @@
+<h2 id="dynamic-summaries">Dynamic summaries</h2>
 <p>
-  Define additional summary sets as described in
-  the <a href="reference/schema-reference.html#summary">schema reference</a>.
+  Use <a href="reference/schema-reference.html#summary">dynamic</a>
+  to generate dynamic abstracts of fields, based on query keywords.
+  Example from Vespa Documentation Search - see the
+  <a href="https://github.com/vespa-cloud/vespa-documentation-search/blob/main/src/main/application/schemas/doc.sd">
+  schema</a>:
+</p>
+<pre>
+document doc {
+
+    field content type string {
+        indexing: summary | index
+        <span class="pre-hilite">summary : dynamic</span>
+    }
+</pre>
+<p>
+  A query for <em>document summary</em> returns:
+</p>
+<blockquote>
+  Use <strong>document summaries</strong> to configure which fields ...
+  indexing: <strong>summary</strong> | index } } <strong>document-summary</strong>
+  titleyear { <strong>summary</strong> title type string ...
+</blockquote>
 <p>
-  Example: the <em>title</em> and <em>year</em> fields are included in a the <em>titleyear</em> summary.
+  The example above creates a dynamic summary with the matched terms highlighted.
+  The latter is called <a href="reference/schema-reference.html#bolding">bolding</a>
+  and can be enabled independently of dynamic summaries.
+</p>
+<p>
+  Refer to the <a href="reference/schema-reference.html#summary">reference</a> for the response format.
+</p>
+
+
+
+<h2 id="summary-classes">Summary classes</h2>
+<p>
+  Use summary classes - <a href="reference/schema-reference.html#summary">reference</a> - to simplify the YQL,
+  by naming the fields in a class:
 </p>
 <pre>
 schema music {
 
     document music {
-        field title type string {
+        field artist type string {
             indexing: summary | index
         }
-        field artist type string {
-            indexing: summary | attribute | index
+        field album type string {
+            indexing: summary | index
+            index: enable-bm25
         }
         field year type int {
             indexing: summary | attribute
         }
-        field popularity type int {
+        field category_scores type tensor&lt;float&gt;(cat{}) {
             indexing: summary | attribute
         }
-        field url type uri {
-            indexing: summary | index
-        }
     }
 
-    document-summary titleyear {
-        summary title type string {
-            source: title
-        }
-        summary year type int {
-            source: year
-        }
-    }
+    <span class="pre-hilite">document-summary content {</span>
+        <span class="pre-hilite">summary artist type string {</span>
+            <span class="pre-hilite">source: artist</span>
+        <span class="pre-hilite">}</span>
+        <span class="pre-hilite">summary album type string {</span>
+            <span class="pre-hilite">source: album</span>
```

Comment:
The source is not needed when it is equal to the summary field name. Consider skipping to simplify the example.